docs/pages/build/eas-builds-in-5-minutes.md

```diff
@@ -0,0 +1,169 @@
+---
+title: EAS Builds in 5 Minutes
+---
+
+Let's get started by building Android and iOS app binaries for a fresh bare project initialized with Expo CLI.
+
+## Prerequisites
+
+- Install Expo CLI by running `npm install -g expo-cli` (or `yarn global add expo-cli`). If you already have it, make sure you're using the latest version. We're constantly improving the experience with EAS Builds, and since it's still in alpha, a slightly older version than the latest can become useless. **This tutorial assumes you're using `expo-cli@3.23.2` or newer.**
+- Sign in with `expo login` or sign up with `expo register` if you don't have an Expo account yet. You can check if you're logged in by running `expo whoami`.
+
+## Initialize a New Project
+
+Run `expo init PROJECT_NAME` (let's assume `PROJECT_NAME = abcd`) and choose a bare workflow template (either `minimal` or `minimal (TypeScript)`).
+
+<center><img src="/static/images/eas-builds/5-minute-tutorial/01-init.png" /></center>
+
+Initializing the project can take a few minutes. Once it's all set, you should see `✅ Your project is ready!` in logs.
+
+<center><img src="/static/images/eas-builds/5-minute-tutorial/02-init-complete.png" /></center>
+
+Expo CLI creates a git repository for you. However, some changes are not commited by default. Go into the project directory, review the changes, and commit them:
+
+- `cd abcd`
+- `git add .`
+- `git commit -m "Add Podfile.lock, yarn.lock, etc."`
+
+<center><img src="/static/images/eas-builds/5-minute-tutorial/03-initial-commit.png" /></center>
+
+## Create eas.json
+
+The next step towards building your native project with EAS Builds is creating the `eas.json` file in the root directory. It will contain the EAS Builds configuration. Create the file with the following contents:
```

Comment:
maybe we should add `expo eas:init` to create this file

Reply:
This sounds like a great idea! https://github.com/expo/turtle-v2/issues/272